src/main/java/com/amazon/opendistroforelasticsearch/knn/index/IndexUtil.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *   Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.knn.index;
+
+import java.io.File;
+
+public class IndexUtil {
+
+    public static Long BYTES_PER_KILOBYTES = 1024L;
+
+    /**
+     * Determines the size of a file on disk in kilobytes
+     *
+     * @param filePath path to the file
+     * @return length of the file in kilobytes
+     */
+    public static long getFileSizeInKB(String filePath) {
+        if (filePath == null || filePath.isEmpty()) {
```

Comment:
This will not differentiate empty file with invalid file path or null. Is this intended?

Reply:
So I guess it would say an empty file has a size of 1 Kb, where as a non-existent file has a size of 0.